package.json

```diff
@@ -88,7 +89,11 @@
   },
   "size-limit": [
     {
-      "path": "dist/index.js",
```

Comment:
Changing `type` to `module` affects the `build` command in `package.json`, such that `index.js` is no longer built to `./dist`, but a common js file (`index.cjs`) and an ES script file (`index.module.js`) are output instead, so new tests are needed to replace this one (see below).